s-nail.yaml

```
@@ -0,0 +1,51 @@
+package:
+  name: s-nail
+  version: 14.9.25
+  epoch: 1
+  description: "S-nail v14.9.25"
```

```suggestion
  description: "S-nail provides a simple and friendly environment for sending and receiving mail."
```

The previous description will get out of date, and this is taken from the upstream repo's readme.

---

Yes, of course, this makes more sense.